Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -281,7 +281,7 @@ class BrowserViewController: UIViewController {
         })
 
 
-        searchLoader = SearchLoader(history: profile.history, urlBar: urlBar)
+        searchLoader = SearchLoader(profile: profile, urlBar: urlBar)
```

Comment:
Yeah, see subsequent bug comment; this isn't a complete fix.

I'm working in follow-up commits to fix some of the worst bugs in `BrowserDB` (like opening one connection per table!), and then to make initialization all occur in ordinary sequenced operations on the SwiftData queue.